src/GapsInCareHelpers.ts

```diff
@@ -344,6 +352,8 @@ export function calculateReasonDetail(
                 }
               });
             }
+          } else if (f.type === 'notnull') {
+            reasonDetail.reasonCodes.push(CareGapReasonCode.VALUEMISSING);
```

Comment:
This doesn't check if the resource is missing a value or not for the specified attribute. I know this may fall into realm of the Reason Detail work that I am starting. The above code on the `detail` filter does deal with checking if the value is in the interval so it may be prudent to do the same type of check here.